test/Vagrantfile

```diff
@@ -34,6 +34,7 @@ $provisionSh = <<-SHELL
 
     sudo yum -y -q update --exclude=puppet*
     echo "Install Hootenanny"
+#     sudo yum -y -v install hootenanny-autostart
```

Comment:
Do we want the verbose output since this a test run?  If it's not useful then I'm fine with the change.